docs/ir/ir-breaking-changes.rst

```diff
@@ -33,3 +33,38 @@ Consequently, if the padding space within a struct is used to store data (e.g. i
     }
 
 We have the same behavior for implicit delete, for example when array of structs is shortened.
+
+ * The order of contract initialization has changed in case of inheritance.
+
+The order used to be:
+ - Evaluate base constructor arguments from most derived to most base contract.
```

Comment:
Shouldn't we also say that all state variables are 0 before this? This is important in case you call internal functions that read state variables

Reply:
Added